### PR TITLE
bugfix/AB#107-Stringlengh

### PR DESCRIPTION
**Pull request by DerCed**
allowing to add max and min to string

## Description
Fixes AB#107

## Motivation and Context
String length should be definable

## How has this been tested?
Local browser

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: webclient/app/src/app/commons/entityDialog/EntityDialog.jsx [hunk 1]
```diff
@@ -63,7 +63,9 @@ function EntityDialog(props) {
     const dataTypes = [
         {
             id: 1,
-            name: "String"
+            name: "String",
+            allowMin: true,
```
Comment (witchpou) [inline webclient/app/src/app/commons/entityDialog/EntityDialog.jsx:67]: die Felder für String sollten auf  minlengh und maxlength gemappt werden...ist das berücksichtigt?           
"fieldValidateRules": [
                "required",
                "minlength",
                "maxlength"
            ],

**Review by witchpou — CHANGES_REQUESTED** [review]

File: webclient/app/src/app/commons/entityDialog/EntityDialog.jsx (hunk 1 above)
Comment (DerCed) [reply webclient/app/src/app/commons/entityDialog/EntityDialog.jsx:67]: Durch die Felder kannst du einstellen, ob min und max eingegeben werden kann. Also jetzt sind die beiden Felder frei und kannst Daten wir bei z.B. Integer eingeben.

Comment (witchpou) [reply webclient/app/src/app/commons/entityDialog/EntityDialog.jsx:67]: das beantwortet die Frage nicht - das mapping sollte so aussehen:
{
    "name": "MyTestEntity",
    "fields": [
        {
            "fieldName": "testField",
            "fieldType": "String",
            "fieldValidateRules": [
                "required",
                "minlength",
                "maxlength"
            ],
            "fieldValidateRulesMinlength": 2,
            "fieldValidateRulesMaxlength": 300
        },
        {
            "fieldName": "testField2",
            "fieldType": "Long",
            "fieldValidateRules": [
                "required",
                "min",
                "max"
            ],
            "fieldValidateRulesMin": 0,
            "fieldValidateRulesMax": 200
        }
    ]
}

**Comment by DerCed** [comment]
@witchpou is this implemented with #255 ? Then I could close this PR

**Pull request closed without merging**